cmd/img_mgmt/src/img_mgmt.c

```diff
@@ -137,21 +137,29 @@ img_mgmt_read_info(int image_slot, struct image_version *ver, uint8_t *hash,
     size_t data_off;
     size_t data_end;
     bool hash_found;
+    uint8_t erased_val;
+    uint32_t erased_val_32;
     int rc;
 
+    rc = img_mgmt_impl_erased_val(image_slot, &erased_val);
+    if (rc != 0) {
+        return MGMT_ERR_EUNKNOWN;
+    }
```

Comment:
Maybe this feature could be made conditional based on syscfg so that old functionality can still be used.

Reply:
What old functionality cannot be used with this feature?

Reply:
Well, the hardcoded value of 0xff, but you are right all hal flash set it to 0xff anyway. So, it is fine.